src/views/blocks/rsvp/form/opt-out.php

```diff
@@ -13,21 +13,25 @@
  * @since 4.9
  * @since 4.11.0 Updated the optout markup and classes used.
  * @since 4.11.3 Ensure we always show the optout by default.
+ * @since TBD Add $post_id to filter for hiding opt-outs.
  *
- * @version 4.11.3
+ * @version TBD
  *
  */
-$modal    = $this->get( 'is_modal' );
-$ticket   = $this->get( 'ticket' );
+$modal   = $this->get( 'is_modal' );
+$ticket  = $this->get( 'ticket' );
+$post_id = $this->get( 'post_id' );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $post_id

<sub>hash: cdad6e1e7405f11b5d0cb4e5ff5705cb</sub>